deployment/CP03ISPM_Deploy.csv

```diff
@@ -26,3 +26,12 @@ AR39,,AR44,,CP03ISPM-WFP01-02-DOFSTK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24
 AR39,,AR44,,CP03ISPM-WFP01-03-CTDPFK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24:00,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40.364817,-70.888668,,68,92,
 AR39,,AR44,,CP03ISPM-WFP01-04-FLORTK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24:00,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40.364817,-70.888668,,68,92,
 AR39,,AR44,,CP03ISPM-WFP01-05-PARADK000,3,1,2019-10-06T13:41:00,2020-06-13T19:24:00,CGMCP-03ISPM-00003,CGVEH-CWFPCP-30001,CGINS-PARADK-20541,40.364817,-70.888668,,68,92,
+AR48,,,,CP03ISPM-RII01-02-ADCPTG010,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,,CGINS-ADCPTG-20496,40,-70,,,,
+AR48,,,,CP03ISPM-SBS01-00-STCENG000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,,CGCON-BSTC01-00018,40,-70,,0,,
+AR48,,,,CP03ISPM-SBS01-01-MOPAK0000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,,CGINS-MOPAK0-58481,40,-70,,0,,
+AR48,,,,CP03ISPM-WFP01-00-WFPENG000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,40,-70,,,,
+AR48,,,,CP03ISPM-WFP01-01-VEL3DK000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,40,-70,,,,
+AR48,,,,CP03ISPM-WFP01-02-DOFSTK000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,40,-70,,,,
+AR48,,,,CP03ISPM-WFP01-03-CTDPFK000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40,-70,,,,
+AR48,,,,CP03ISPM-WFP01-04-FLORTK000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40,-70,,,,
+AR48,,,,CP03ISPM-WFP01-05-PARADK000,4,1,2020-10-28T00:00:00,,CGMCP-03ISPM-00004,CGVEH-CWFPCP-30001,CGINS-PARADK-20608,40,-70,,,,
```

Comment:
Looks good. Confirmed SNs with config sheet

Reply:
Looks good.  Compared to Config Sheet, WFP QCT 3310-00100-00126, and WFP instrument cal docs.